### DISCUSSION TITLE
Log entries not showing

### DISCUSSION THREAD

**Discussion by SethPowsner** [magicbug/Cloudlog#1738] [Q&A] [answered]
None of my old entries appeared Sunday when I logged-in to add a QSO. I'm subscribed to the web cloud logging service. Haven't made any entries since the first half of August.

Clicking Logbook lead to the error message "No logbooks were found. You need to define a logbook under Station Logbooks!"

Did I miss some switch-over / reconfiguration?
Sorry if I've messed up something obvious or this isn't the right place to ask about this.

Thanks
Seth W1ETA

**Comment by magicbug** [comment, marked as answer]
You need to create a station logbook via the callsign dropdown menu top right then link some locations to it and it will all return to normal :)

**Reply by SethPowsner** [reply]
That was it. All entries were there. Thanks.

**Comment by SethPowsner** [comment]
OK! Working. Even registered the QSO from the weekend that I thought was lost.
Thanks!

On Oct 31, 2022, at 1:37 PM, Peter Goodhall ***@***.***> wrote:


You need to create a station logbook via the callsign dropdown menu top right then link some locations to it and it will all return to normal :)

**Comment by stormadvisor** [comment]
@magicbug , So I have done this and I am not getting anything but the message
![Screenshot from 2023-06-19 10-31-05](https://github.com/magicbug/Cloudlog/assets/62911574/80e76548-f9fe-4f7f-85bd-665469013cdd)
I have imported and created new and still nothing
![Screenshot from 2023-06-19 10-31-59](https://github.com/magicbug/Cloudlog/assets/62911574/fcf4e499-fb99-44f2-85e0-d90018a7ab61)
I am obviously doing something wrong. Any ideas?
